src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -84,6 +84,7 @@ protected function configure()
                 new InputOption('domain', null, InputOption::VALUE_OPTIONAL, 'Specify the domain to update'),
                 new InputOption('xliff-version', null, InputOption::VALUE_OPTIONAL, 'Override the default xliff version', '1.2'),
                 new InputOption('sort', null, InputOption::VALUE_OPTIONAL, 'Return list of messages sorted alphabetically', 'asc'),
+                new InputOption('as-tree', null, InputOption::VALUE_OPTIONAL, 'Dump the messages as a tree-like structure: The given value defines the level where to switch to inline YAML'),
```

Comment:
why not set the default value here?

Reply:
I wasn't sure about that at first, but decided to remove the default value because I can just call `...'as_tree' => $input->getOption('as-tree')...` when setting the writers options which will result in `'as_tree' => null` and with this make sure that the `YamlFileDumper` wouldn't call the `ArrayConverter::expandToTree($data);` method every time.